packages/editor-ui/src/api/templates.ts

```diff
@@ -6,6 +6,7 @@ import type {
 	ITemplatesCollectionResponse,
 	ITemplatesWorkflowResponse,
 	IWorkflowTemplate,
+	TemplateSearchFacet,
```

Comment:
Should we drop the endpoint to retrieve the categories since we are not using it anymore? @MiloradFilipovic

Reply:
Yes